exercises/practice/after-party/after_party_test.py

```diff
@@ -0,0 +1,95 @@
+# These tests are auto-generated with test data from:
+# https://github.com/exercism/website/blob/main/bootcamp_content/projects/string-puzzles/exercises/after-party/config.json
+# File last updated on 2024-02-28
+
+import unittest
+from after_party import (
+    on_guest_list,
+)
+
+
+class AfterPartyTest(unittest.TestCase):
+    def test_empty_list(self):
+        """
+        slug: empty-list
```

Comment:
The slug shows up in the unit test function name. It doesn't need repeating. The description could be used to craft a docstring summary sentence.